src/sentry/grouping/fingerprinting/__init__.py

```diff
@@ -229,6 +229,9 @@ def from_json(cls, value, bases=None):
 
     @staticmethod
     def from_config_string(s, bases=None):
+        if not s:
```

Comment:
This is required because the grammar fails if you pass an `""` string, maybe due to the fact that version `0.10.0` mentions some fixes for `*` and `+` quantifiers (https://github.com/erikrose/parsimonious/compare/0.9.0...0.10.0#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR451)